orgs/orgs.yml

```diff
@@ -218,6 +218,9 @@ orgs:
         description: BOSH CLI v2+
         has_projects: false
         has_wiki: false
+      bosh-common:
+        default_branch: main
+        has_projects: true
```

Comment:
why do you need projects?

Reply:
Because I copy-pasted from other sections. Happy to remove (or have someone remove it)

Reply:
All good. I will check on it and correct if need.